apps/condo/domains/billing/schema/BillingReceiptFile.js

```diff
@@ -68,8 +115,19 @@ const BillingReceiptFile = new GQLListSchema('BillingReceiptFile', {
         auth: true,
     },
     hooks: {
-        // Set meta to file for checking permissions { listkey, id }
-        afterChange: fileMetaAfterChange,
+        afterChange: async ({ updatedItem, listKey }) => {
+            if (updatedItem && Adapter.acl && Adapter.acl.setMeta) {
+                const sensitiveFile = get(updatedItem, 'sensitiveDataFile.filename')
+                const publicFile = get(updatedItem, 'publicDataFile.filename')
```

Comment:
Lodash always casts a string path to an array. So we can decrease the number of calculations.
```suggestion
                const sensitiveFile = get(updatedItem, ['sensitiveDataFile', 'filename'])
                const publicFile = get(updatedItem, ['publicDataFile', 'filename'])
```

Reply:
it's hard to use such syntax on constructions like
```
const residentName = get(residents, '[0].name')
```
So, I prefer string variant